### ISSUE TITLE
Android Camera Demo: Android NDK r11 can't be compiled by bazel after modified the WORKSPACE

### ISSUE THREAD

**Issue by waterofphysics** [tensorflow/tensorflow#1468]
GitHub issues are for bugs / installation problems / feature requests.  
For general support from the community, see [StackOverflow](https://stackoverflow.com/questions/tagged/tensorflow).
To make bugs and feature requests more easy to find and organize, we close issues that are deemed
out of scope for GitHub Issues and point people to StackOverflow.

For bugs or installation issues, please provide the following information.
The more information you provide, the more easily we will be able to offer
help and advice.

### Environment info
Operating System:
OS X EI Capitan version 10.11.3

If installed from sources, provide the commit hash:
13ea3ca91ba5aecab6f21acc14b9cb6a9afa8630
### Steps to reproduce
1. Build TensorFlow without uncommenting the Android SDK and NDK parts successfully.
2. We can 
```
import tensorflow 
```
in python environment after step 1 and following installation steps.
4. Download SDK and NDK from what https://github.com/tensorflow/tensorflow/blob/master/tensorflow/examples/android/README.md suggested.
3. Uncomment the Android SDK and NDK repository in WORKSPACE and change the path.
4. Rebuild using 
```
$ bazel build -c opt //tensorflow/tools/pip_package:build_pip_package
```
the console returns the error,
```shell
ERROR: .../tensorflow/WORKSPACE:10:1: no such package '@androidndk//': Could not read RELEASE.TXT in Android NDK: /private/var/tmp/.../49fff0428.../external/androidndk/ndk/RELEASE.TXT (No such file or directory) and referenced by '//external:android/crosstool'.
```
### What have you tried?
1. Commenting NDK repository and leave the SDK in TensorFlow WORKSPACE, build using 
```
$ bazel build -c opt //tensorflow/tools/pip_package:build_pip_package
```
passed and without error.

2.Finding that Downloaded NDK and NDK in /private/var/tmp/.../49fff0428.../external/androidndk/ndk are the same. And there are no RELEASE.TXT in in directory.

### Logs or other output that would be helpful
(If logs are large, please upload as attachment).
```shell
ERROR: .../tensorflow/WORKSPACE:10:1: no such package '@androidndk//': Could not read RELEASE.TXT in Android NDK: /private/var/tmp/.../49fff0428.../external/androidndk/ndk/RELEASE.TXT (No such file or directory) and referenced by '//external:android/crosstool'.
```

Is this error due to the codes or my building processes?

**Comment by moonboots** [comment]
I would try downgrading to the previous version of Android NDK (r10e). Here's the Mac OS X link: https://dl.google.com/android/ndk/android-ndk-r10e-darwin-x86.bin

I ran into similar issues using the latest version of Android NDK (r11) which no longer contains `RELEASE.TXT`. Manually creating that file with `echo r11 > RELEASE.TXT` fixed the immediate problem, but I ran into issues of bazel hard dependencies on r10e.

For linux users, here are the r10e 32/64 links, no longer listed on the Android NDK but still accessible:
https://dl.google.com/android/ndk/android-ndk-r10d-linux-x86_64.bin
https://dl.google.com/android/ndk/android-ndk-r10d-linux-x86.bin
(Thanks [Arch AUR](https://aur.archlinux.org/cgit/aur.git/commit/?h=android-ndk&id=988b6411ee099a762adf981004af86616fabedae))

**Comment by waterofphysics** [comment]
@moonboots 
Thank you for your detailed answers, it really helps me. 
Android NDK (r10e) link for Mac OS X is: 
https://dl.google.com/android/repository/android-ndk-r10e-darwin-x86_64.zip
We should also install Android SDK build tools version 23.0.1 via Android SDK manager to build the android example.
